packages/odyssey-react-mui/src/Checkbox.tsx

```diff
@@ -145,7 +158,15 @@ const Checkbox = ({
 
   return (
     <FormControlLabel
-      sx={{ alignItems: "flex-start" }}
+      sx={{
+        alignItems: "flex-start",
+        ...(isReadOnly && {
+          cursor: "default", //area between the label and checkbox
+          "& .MuiTypography-root": {
+            cursor: "default", //for the label
+          },
+        }),
+      }}
```

Comment:
Can we memoize these styles?

Reply:
I think so, I'll update it to something like this:

  const checkboxStyles = useMemo(
    () => ({
      alignItems: "flex-start",
      ...(isReadOnly && {
        cursor: "default",
        "& .MuiTypography-root": {
          cursor: "default",
        },
      }),
    }),
    [isReadOnly],
  );

  return (
    <FormControlLabel
      sx={checkboxStyles}
...rest...